index.js

```diff
@@ -0,0 +1,56 @@
+window.document.querySelector('#note-form').addEventListener('submit', function(event) {
```

Comment:
- [ ] Extract this element `window.document.querySelector('#note-form')` into a constant